mithril.js

```diff
@@ -218,9 +218,34 @@
 		return list
 	}
 
+	var applyIEPerf = (
+			typeof document !== "undefined" &&
+			typeof document.documentMode === "number"
+		) ||
+		(
+			typeof navigator !== "undefined" &&
+			typeof navigator.userAgent === "string" &&
+			/\bEdge\/\d/.test(navigator.userAgent)
+		)
+
 	function insertNode(parentElement, node, index) {
-		parentElement.insertBefore(node,
-			parentElement.childNodes[index] || null)
+		if (applyIEPerf) {
+			if (!parentElement._children) {
+				parentElement._children = []
+			}
+			parentElement._children.push(node)
```

Comment:
I am still new to this library and have started using this in our big framework at work.
This is where i am looking for some feedback on how things work inside mihtril. The examples I have tried it on works great.

Also its not a small perf difference, its big on IE. https://github.com/spicyj/innerhtml-vs-createelement-vs-clonenode